onnxruntime/server/server_configuration.h

```diff
@@ -43,6 +43,7 @@ class ServerConfiguration {
   unsigned short http_port = 8001;
   int num_http_threads = std::thread::hardware_concurrency();
   onnxruntime::logging::Severity logging_level{};
+  unsigned short grpc_port = 50051;
```

Comment:
https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Dynamic,_private_or_ephemeral_ports

Reply:
demo's default to that one:
https://grpc.io/docs/tutorials/basic/c/